cmd/sonarExecuteScan.go

```diff
@@ -195,6 +218,14 @@ func runSonar(config sonarExecuteScanOptions, client piperhttp.Downloader, runne
 		log.Entry().WithError(err).Warning("no scan report found")
 		return nil
 	}
+
+	var serverUrl string
+
+	if len(config.ServerURL) > 0 {
+		serverUrl := config.ServerURL
```

Comment:
I think = is sufficient here
```suggestion
		serverUrl = config.ServerURL
```

Reply:
Yeah, I have already removed it :)